### DISCUSSION TITLE
USB-C noise

### DISCUSSION THREAD

**Discussion by vurguuz** [Builty/TonexOneController#32] [Q&A]
While waiting for the boards to arrive, I have experimented and noticed that the ToneX One noise floor is at least 7db higher when powered from the USB-C port vs. a 9V isolated PSU. It is as noisy when using both the 9V power and the USB-C port together as when using only the USB-C port.

I'm thinking of taking the 9V isolated power and regulating it down to 5V when making the board, but I'm not sure if it will actually help with the noise issue. 

Have you noticed this?

**Comment by Builty** [comment]
I haven't noticed any noise myself. I have never tried the Tonex via 9v though. I'm also using an FRFR that is not set very high in volume (just playing at home.)
A lot of people have this going now, and so far nobody has mentioned it being an issue.

I have noticed though if I connect more pedals to the same 9v supply (I have the 4.3B on my pedal board) then it gets noisy. 
An isolated supply on the 9v side certainly is worthwhile, but not sure of benefit if its regulated down to 5v.

**Comment by waltiti17** [comment]
> While waiting for the boards to arrive, I have experimented and noticed that the ToneX One noise floor is at least 7db higher when powered from the USB-C port vs. a 9V isolated PSU. It is as noisy when using both the 9V power and the USB-C port together as when using only the USB-C port.
> 
> I'm thinking of taking the 9V isolated power and regulating it down to 5V when making the board, but I'm not sure if it will actually help with the noise issue.
> 
> Have you noticed this?

No noise here too. 

make sure is a 9v with at least 2000ma  or 2A, tonex needs 1200ma and you need to give power to Tonex 1200ma and Waveshare ????ma. but with 2000ma works well. 

Always make sure the Power supply have more ma or A, also with regular pedals or multi-effects pedalboards.

**Comment by vurguuz** [comment]
> I have noticed though if I connect more pedals to the same 9v supply (I have the 4.3B on my pedal board) then it gets noisy.
> An isolated supply on the 9v side certainly is worthwhile, but not sure of benefit if its regulated down to 5v.

I am planning to use the `Waveshare ESP32-S3 Zero` board, which means 5V input, as I understand.

> tonex needs 1200ma and you need to give power to Tonex 1200ma and Waveshare ????ma.

That's kind of strange, [IK's specs](https://www.ikmultimedia.com/products/tonexone/#specs) list the current draw of the ToneX One at 120mA @ 9V, which could roughly translate to 220mA @ 5V. The Zero on the other hand can draw as much as [0.5A](https://www.waveshare.com/wiki/ESP32-S3-Zero). This is really not the issue though, I'm using a 4.5A 5V supply.

I got the 7db figure in the original message while looking at the signals with volume boost applied by my DAW, which (looking back) was probably an incorrect way to measure.

I guess the word "noisy" is exaggerated; feeding power through USB is quiet enough, but a little noisier than when running at 9V.

Thanks for your feedback, guys.

**Comment by Echirouze** [comment]
what's really about with USB C power via waveshare 4.3", i have no choice with the Usb C, the tonex in connected with the waveshare, but i noticed on my computer i have this noise too with the Usb C power, but not with a classic power 9v

**Reply by one1ton** [reply]
Hi,
Maybe slightly off topic: I Did not experience any Noise with the JC3248w535 running only the Tonex One and the controller, I did however have some noise coming from a Swiff C20 tuner powered from the same USB-port. I have finally been able to get rid of the noise issue by iserting a Joyo ZGP-W before the tuner. This device converts 5V to 9V and provides galvanic isolation. I highly recommend this device since it does step-up-transformation as well as noise cancellation and is small and reasonably priced.

Best regards,
Tony

**Reply by Builty** [reply]
This reminds me, on my pedal board I have a Digitech Drop pedal. If I power my Waveshare 4.3 off the same 9v as the Digitech, it gets huge noise on the Tonex. If I power the Digitech on a different 9v, then it is quiet.
